Sanic instructions initialize in before_server_start (#8422)
getsentry/sentry-docs: src/platforms/python/integrations/sanic/index.mdx

If you have the `sanic` package in your dependencies, the Sanic integration will be enabled automatically when you initialize the Sentry SDK.

> To ensure that errors occuring in background tasks get reported to Sentry, we always recommend enabling the [`AsyncioIntegration`](/platforms/python/integrations/asyncio/) when you are using the `SanicIntegration`. To enable the `AsyncioIntegration`, you must explicitly add the `AsyncioIntegration` to the `integrations` list and initialize the SDK within a `before_server_start` listener.

Reviewer: ```suggestion
To ensure that errors that occur in background tasks get reported to Sentry, enable the [`AsyncioIntegration`](/platforms/python/integrations/asyncio/) when you use the `SanicIntegration`. To enable the `AsyncioIntegration`, you must explicitly add the `AsyncioIntegration` to the `integrations` list and initialize the SDK within a `before_server_start` listener.
```

Author: Thank you for the suggestion, but I think it would be better to keep the initial wording (or something closer to it) here, since I would like to emphasize that we recommend users to always initialize the AsyncioIntegration when using the SanicIntegration, since there is no real downside to doing so. With your proposed change, it reads more like we might only be recommending enabling the AsyncioIntegration when there are background tasks that might raise errors
